de.dlr.sc.virsat.model.extension.requirements/src/de/dlr/sc/virsat/model/extension/requirements/model/Requirement.java

```diff
@@ -52,4 +55,34 @@ public Requirement(Concept concept) {
 	public Requirement(CategoryAssignment categoryAssignment) {
 		super(categoryAssignment);
 	}
+	
+	/**
+	 * Update the requirement name to the name of its identifying attributes
+	 * @param ed the editing domain
+	 * @return the command
+	 */
+	public Command updateNameFromAttributes(EditingDomain ed) {
+		return setName(ed, getNameFromAttributes());
+	}
+	
+	/**
+	 * Update the requirement name to the name of its identifying attributes
+	 */
+	public void updateNameFromAttributes() {
+		setName(getNameFromAttributes());
+	}
+	
+	/**
+	 * Create a name from all identifying attributes
+	 * @return the name derived from the attributes
+	 */
+	protected String getNameFromAttributes() {
+		String newReqName = "Req";

```

Comment:
I believe I have seen the Req prefix in some other class as static constant.
Maybe we should unify them. Also in the test cases the Req prefix seems to be redeclared.